parser_ua.go

```diff
@@ -0,0 +1,77 @@
+package whoisparser
+
+import (
+	"regexp"
+)
+
+var uaParser = &Parser{
+	errorRegex: &ParseErrorRegex{
+		NoSuchDomain:     regexp.MustCompile(`No entries found for`),
+		RateLimit:        nil,
+		MalformedRequest: regexp.MustCompile(`Unimplemented object service`),
+	},
+
+	registrarRegex: &RegistrarRegex{
+		DomainName:     regexp.MustCompile(`domain: *(.*)`),
+		NameServers:    regexp.MustCompile(`nserver: *(.*)`),
+		DomainStatus:   regexp.MustCompile(`(?s)nserver:.*?status: *(.*?)\ncreated`),
+		CreatedDate:    regexp.MustCompile(`domain:(?:.*\s)*?created: *(.*)`),
+		UpdatedDate:    regexp.MustCompile(`domain:(?:.*\s)*?modified: *(.*)`),
+		ExpirationDate: regexp.MustCompile(`domain:(?:.*\s)*?expires: *(.*)`),
+		Emails:         regexp.MustCompile(`abuse-email: *(.*)`),
+		RegistrarName:  regexp.MustCompile(`registrar: *(.*)`),
+	},
+	techRegex: &RegistrantRegex{
+		// (?:[^n][^\\][^a]|[^n][^\\]a|[^n]\\a|[^n]\\[^a]|n\\[^a]|n[^\\][^a]) - negative lookahead alternative
```

Comment:
Is this comment necessary?

Reply:
Only for explanation of this regex functional.

Reply:
If something requires any explanation please create variable with corresponding clear name. Comments should explain intentions not implementation details.

Reply:
Done